src/applications/disability-benefits/view-payments/tests/helpers/index.unit.spec.js

```diff
@@ -43,7 +43,7 @@ describe('helper functions', () => {
   describe('formatDate', () => {
     context('when given a valid date in the correct format', () => {
       it('returns a date in the correct format', () => {
-        const inputs = ['2019-04-12T00:00:00.000-06:00', '2019-04-12'];
```

Comment:
was there a reason you changed the time here?

Reply:
So that this test would fail for anyone running the test in the US. The way it was it would only fail if the users computer was set to Pacific Time

Reply:
Yes it does, the test was not failing with the old code unless the users time zone was Pacific Time. Changing it from -6 to -3 should make it so that running this test with the old code will fail for all US time zones